### ISSUE TITLE
There are error in callbacks function click button

### ISSUE THREAD

**Issue by edorozco** [shiv19/nativescript-cfalert-dialog#1]
Clicks button on `dialogStyle: CFAlertStyle.BOTTOM_SHEET,` never return your repective callback click button, always return the last callback button that was setup.

```javascript
{
	"text": "Okay",
	"buttonStyle": CFAlertActionStyle.POSITIVE,
	"buttonAlignment": CFAlertActionAlignment.JUSTIFIED,
	"onClick": function(response) {
		console.log(response);
	}
}, {
	"text": "Nope",
	"buttonStyle": CFAlertActionStyle.NEGATIVE,
	"buttonAlignment": CFAlertActionAlignment.JUSTIFIED,
	"onClick": function(response) {
                // always return here, because is the last button in array
		console.log(response);
	}
}
```

Ex. if I tapping OK, enter into callback of Nope Function.

**Comment by shiv19** [comment]
I see. Will take a look into it today. Thanks for finding the bug.
@edorozco

**Comment by shiv19** [comment]
@edorozco
I have confirmed the issue.
Will fix it by the end of the day

**Comment by edorozco** [comment]
Thanks dude.! Im wait for it

**Comment by shiv19** [comment]
@edorozco 
Issue resolved in version 1.0.5 😄  enjoy 👍

**Comment by edorozco** [comment]
Thanks dude, nice job.. its works ok
